core/client/fs/src/main/java/alluxio/client/file/MetadataCachingBaseFileSystem.java

```diff
@@ -50,6 +51,7 @@
   private final MetadataCache mMetadataCache;
   private final ExecutorService mAccessTimeUpdater;
   private final boolean mDisableUpdateFileAccessTime;
+  private static final URIStatus NOT_FOUND_STATUS = new URIStatus(new FileInfo());
```

Comment:
nit: move this `NOT_FOUND_STATUS` up with other static final members?

Reply:
Done.